electricpy/__init__.py

```diff
@@ -4782,6 +4782,102 @@ def wireresistance(length=None,diameter=None,rho=16.8*10**-9,R=None):
     elif R != None and length != None:
         A = rho*length/R
         return _np.sqrt(4*A/pi)
+
+def parallel_plate_capacitance(A=None, d=None, e=e0, C=None):
+    r"""
+    Parallel-Plate Capacitance Calculator.
+
+    Enter three values to calculate the remaing one. Even though every variable
+    is unitless, please use the International System of Units.
+
+    .. math:: C =\frac{\varepsilon A}{d}
```

Comment:
consider using `\cdot` to denote multiplication in the numerator, it makes things explicitly clear that there's multiplication taking place